composer.json

```diff
@@ -9,10 +9,7 @@
         "silverstripe/recipe-cms": "5.x-dev",
         "silverstripe/blog": "4.x-dev",
         "silverstripe/widgets": "3.x-dev",
-        "silverstripe/content-widget": "3.x-dev",
         "silverstripe/spamprotection": "4.x-dev",
```

Comment:
Is there any point having spam protection in here if comments are not there.

Reply:
Probably not, though spam protection is still supported so it should stay there